includes/helper/class-wp-job-manager-helper.php

```diff
@@ -468,7 +468,7 @@ private function activate_licence( $product_slug, $licence_key, $email ) {
 	private function deactivate_licence( $product_slug ) {
 		$licence = $this->get_plugin_licence( $product_slug );
 		if ( empty( $licence['licence_key'] ) || empty( $licence['email'] ) ) {
-			$this->add_error( $product_slug, __( 'licence is not active.', 'wp-job-manager' ) );
+			$this->add_error( $product_slug, __( 'license is not active.', 'wp-job-manager' ) );
```

Comment:
Good catch. Comments and option names use licence (WPJM's original dev was a Brit ;)) but all strings should use US spelling (license).